### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Make it easier to MT (machine translate). Conform to MS style guide such as Avoid **_please_** except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation.

I let one please slide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ## Welcome to dotnet sdk
 
-This repo contains core functionality needed to create .NET projects that is shared between VisualStudio and CLI.
+This repository contains core functionality needed to create .NET projects that is shared between Visual Studio and the [].NET CLI](https://learn.microsoft.com/dotnet/core/tools/).
```
Comment (Rick-Anderson) [inline README.md:3]:
```suggestion
This repository contains core functionality needed to create .NET projects that is shared between Visual Studio and the [.NET CLI](https://learn.microsoft.com/dotnet/core/tools/).
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+This repository contains core functionality needed to create .NET projects that is shared between Visual Studio and the [.NET CLI](https://learn.microsoft.com/dotnet/core/tools/).
 
-* MSBuild tasks can be found under [/src/Tasks/Microsoft.NET.Build.Tasks/](src/Tasks/Microsoft.NET.Build.Tasks).
+* MSBuild tasks are under [/src/Tasks/Microsoft.NET.Build.Tasks/](src/Tasks/Microsoft.NET.Build.Tasks).
 
-Please refer to [dotnet/project-system](https://github.com/dotnet/project-system) for the project system work that is specific to Visual Studio.
+See [dotnet/project-system](https://github.com/dotnet/project-system) for the project system work that is specific to Visual Studio.
 
-This repo also contains [common project and item templates](https://github.com/dotnet/sdk/tree/main/template_feed).
+This repository also contains [common project and item templates](https://github.com/dotnet/sdk/tree/main/template_feed).
```
Comment (Rick-Anderson) [inline README.md:9]:
```suggestion
Common project and item templates are found in [template_feed](https://github.com/dotnet/sdk/tree/main/template_feed).
```

**Comment by Rick-Anderson** [comment]
@mkArtakMSFT can you recommend a reviewer?

**Review by baronfel — APPROVED** [review]

**Comment by baronfel** [comment]
Thanks for the style fixes, @Rick-Anderson :+1:

**Pull request merged**